src/elisp/treemacs-impl.el

```diff
@@ -886,10 +885,13 @@ PROJECT Project Struct"
     (treemacs-goto-file-node path project))
    ((eq :custom (car path))
     (treemacs--goto-custom-top-level-node path))
-   (t
-    (treemacs--goto-custom-dir-node path))))
+   ((stringp (car path))
+    (treemacs--goto-custom-dir-node path))
+   (t 
```

Comment:
There's a trailing whitespace here. I recommend `(add-hook 'before-save-hook #'delete-trailing-whitespace)`.

Reply:
Oops, my `.emacs.d` was missing `.editorconfig`. I hope the other PRs don't have trailing WS as well...